SU-HSDO/suhumsci
# STN-505 Update utility classes to work with Drupal Views

# READY FOR REVIEW

## Summary
Update the following utility classes to also work in view fields:

- hs-caption
- hs-credits
- hs-emphasized-text
- hb-descriptor
- hb-blockquote
- hb-title
- hb-highlighted-label
- hb-single-category

This required both creating a reset for text for some classes to be able to take effect as well as creating more specificity to some classes so they would apply correctly within Drupal Views fields.

Jira Card: [STN-505](https://sparkbox.atlassian.net/browse/STN-505)

## Steps to Test
1. npm run test
2. You can use these two views to test these classes pretty effectively:
- http://sparkbox-sandbox.suhumsci.loc/news
- http://sparkbox-sandbox.suhumsci.loc/people-0
3. In order to test these classes you'll need to apply them to the views fields:
- If you click on the contextual link within the View you'd like to edit, you'll click "edit view". 
- Once you are inside the view edit you'd like to test, if you click on the view field you want to test, such as "Title" you'll be able to add class styles within the "Style Settings" option.
![image](https://user-images.githubusercontent.com/9355698/92292538-b0d9a400-eeeb-11ea-91df-9b8adc1e4fd0.png)
4. You can test the style classes in either the field class or adding a label class. The styles should apply to either as expected.

References:
This page also lists some of these classes and can be used as a visible reference if needed: http://sparkbox-sandbox.suhumsci.loc/old-stuffutilities/field-utilities

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
